exec/java-exec/src/test/java/org/apache/drill/exec/store/parquet/TestParquetFilterPushDown.java

```diff
@@ -138,18 +138,25 @@ public void testIntPredicateWithEval() throws Exception {
     testParquetRowGroupFilterEval(footer, "intCol = 50 or intCol = 160", RowsMatch.SOME);
 
     //"nonExistCol" does not exist in the table. "AND" with a filter on exist column
+
+    //Ignored till run-time row group pruning is added.
```

Comment:
See my comment about handling non-existent columns.

Reply:
@dvjyothsna / @amansinha100 not sure that we should allow ignoring tests. Does it mean that we are breaking existing functionality?

Reply:
I believe these tests should be enabled now .. it might be an omission.  there are couple of new unit tests for non-existing columns in TestParquetMetadataCache.

Reply:
I'm re-enabling these tests back. All the handling of non-existent columns works fine now.